lib/mix/test/mix/tasks/profile.tprof_test.exs

```diff
@@ -77,15 +77,23 @@ defmodule Mix.Tasks.Profile.TprofTest do
     in_tmp(context.test, fn ->
       assert capture_io(fn ->
                Tprof.run(["--sort", "calls", "-e", @expr])
-             end) =~ ~r/Enum\.each\/2.*String\.Chars\.Integer\.to_string\/1/s
+             end) =~ ~r/\nEnum\.each\/2.*\nString\.Chars\.Integer\.to_string\/1/s
```

Comment:
Note: prepending `\n` here because otherwise it might match due to `anonymous fn/3 in Enum.each/2`